scripts/establish_conda_env.sh

```diff
@@ -146,6 +155,15 @@ function create_libraries()
     if [[ "$PROXY_COMM" != "" ]]; then COMMAND=`echo $COMMAND --proxy $PROXY_COMM`; fi
     if [[ $ECE_VERBOSE == 0 ]]; then echo ...pip-only command: ${COMMAND}; fi
     ${COMMAND}
+    # pyomo only
+    if [[ $ECE_VERBOSE == 0 ]]; then echo ... Installing libraries from pyomo ...; fi
+    local COMMAND=`echo $($PYTHON_COMMAND ${RAVEN_LIB_HANDLER}  ${INSTALL_OPTIONAL} ${OSOPTION} conda --action install --subset pyomo)`
+    if [[ $ECE_VERBOSE == 0 ]]; then echo ... pyomo command: ${COMMAND}; fi
+    if [[ ${COMMAND} == *"pyomo-extensions"* ]];
+    then
+      pyomo download-extensions
+      pyomo build-extensions || echo "Pyomo build failed"
```

Comment:
@joshua-cogliati-inl Could you comment on the modifications in this file "establish_conda_env.sh"?

Reply:
They look fine to me, though it might be worth adding a comment that this should be removed when pyomo starts creating a pip package for pynumero.

Reply:
Added comment.